nvml/manifest.json

```
@@ -1,22 +1,22 @@
 {
-  "display_name": "nvml",
+  "display_name": "Nvidia NVML",
```

```suggestion
  "display_name": "nvml",
```
We can't change the `display_name`, but changing the `public_title` should be enough for this use case